.github/workflows/CI.yml

```diff
@@ -77,7 +81,17 @@ jobs:
       - name: Run tests
         run: npm test
       - name: Run examples
-        run: npm run examples
+        run: SCYLLA_URI=172.42.0.2:9042 npm run examples
```

Comment:
It is recommended to pass all 3 nodes as contact points to the driver, to make tests more realistic. This way it's always the first node who hosts the control connection; it's better to allow more varied scenarios.

Reply:
Currently the examples don't use more than one node. I left creating three Scylla nodes as is in rust driver, as it is very likely we will add more examples/tests that use more than one node in the future.

Reply:
What do you mean by using more nodes? Scylla, being a distributed system, uses all nodes transparently to serve requests.

Reply:
Even if examples only provide one initial contact point (address of one node) to the driver, the driver soon recognizes all nodes and contacts them.

Reply:
Ok. I didn't know that. So then the current examples just uses single entrypoint. My goal with this PR was just to only add those examples to CI and we can add more examples showing things like multiple entry points alongside developing the driver.